core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -137,14 +137,8 @@ public void update(float delta) {
      * @param batch the batch used to draw with
      */
     public void draw(SpriteBatch batch) {
-        //batch.draw(cannon_texture, hitbox.x, hitbox.y, BALL_RAD, BALL_RAD);
-
-        batch.draw(cannonTextureRegion, xpos - width / 2f - 3, ypos - height / 2f + 23, 
-                width / 2f, height / 2f, width, height, 5 / 2, 1, angle, true);
-
-        //batch.draw(cannon_texture_region, x - width / 2f, y - height / 2f,    
-        //width / 2f, height / 2f, width, height, 1, 1, angle, true);
-
+        batch.draw(cannonTextureRegion, xpos - width / 2f + 11, ypos - height / 2f + 7, 
```

Comment:
Agree. I tried changing the sprite shape to another shape (that's why this part took some extra time with me) but couldn't get a design that's centered. Might follow the transparent pixels idea of yours.
Will check and get back with a commit later.